File: Fibonacci.cs
using System;
using System.Collections.Generic;
using System.IO;
class Solution {
    static void Main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
        var line1 = System.Console.ReadLine().Trim();
        var n = Int32.Parse(line1);
        Console.WriteLine(Fib(n));
    }
    
    public static int Fib(int n) {
        if (n < 2) {
            return n;
        }
        
        return (Fib(n-1) + Fib(n-2));
    }
}

File: Fibonacci.js
function processData(input) {
    //Enter your code here
    var n = parseInt(input);
    console.log(fibonacci(input));
} 

function fibonacci(n) {

    if(n < 2) {
        return(n);
    }
    
    return (fibonacci(n-1) + fibonacci(n-2));
}